squadutilslib.py

```diff
@@ -193,20 +237,11 @@ def get_reproducer(
         # In theory there should only be one of those
         logger.debug(f"Testrun id: {testrun.id}")
 
-        try:
-            if local:
-                reproducer = get_file(
-                    f"{testrun.job_url}/reproducer", filename=filename
-                )
-            else:
-                reproducer = get_file(
-                    f"{testrun.job_url}/tuxsuite_reproducer", filename=filename
-                )
-        except HTTPError:
-            logger.error(f"Reproducer not found at {testrun.job_url}!")
-            raise ReproducerNotFound
+        reproducer, is_test_reproducer = get_reproducer_from_testrun(
+            testrun.id, filename, local
+        )
         return (
-            Path(reproducer).read_text(encoding="utf-8"),
+            reproducer,
             Build(getid(testrun.build)).metadata.git_describe,
```

Comment:
all the changes in this file can go into the same commit.

Reply:
Done